dask_expr/_expr.py

```diff
@@ -2603,6 +2603,56 @@ def _fusion_pass(expr):
     return expr
 
 
+class Fill(MapOverlap):
+    _parameters = [
+        "frame",
+        "how",
+        "axis",
+        "limit",
+    ]
+    _defaults = {"axis": 0, "limit": None}
```

Comment:
Let's remove axis for now

Reply:
Gone https://github.com/dask-contrib/dask-expr/pull/454/commits/1129903c787c0d19a0ae00513067642809adeafa